package.json

```diff
@@ -25,12 +25,16 @@
     "next:check-types": "yarn workspace @se-2/nextjs check-types",
     "postinstall": "husky install",
     "precommit": "lint-staged",
-    "vercel": "yarn workspace @se-2/nextjs vercel",
-    "vercel:yolo": "yarn workspace @se-2/nextjs vercel:yolo"
+    "vercel:yolo": "vercel --build-env NEXT_PUBLIC_IGNORE_BUILD_ERROR=true"
   },
   "packageManager": "yarn@3.2.3",
   "devDependencies": {
+    "@types/node": "^18.15.3",
```

Comment:
Are we really need it? It seems everything works without it

Reply:
Tysm !! yes yes we can remove it, it was added here [8f19b21](https://github.com/scaffold-eth/se-2/pull/231/commits/8f19b21cd837aaa9bc48927d74942dcece9940da) and since now we are not using `.env` at root we can completely remove it